API/common/exception.py

```diff
@@ -1,25 +1,75 @@
-class ParameterException(Exception):
-    def __init__(self, msg: str) -> None:
-        super().__init__()
-        self.msg = msg
+from common.util import generate_response
 
 
 class CopilotException(Exception):
+    ERROR_CODE = 501
+
     def __init__(self, msg: str) -> None:
         super().__init__()
         self.msg = msg
 
+    def to_response_body(self) -> str:
+        return generate_response([], self.ERROR_CODE, self.msg)
+
+
+class RequestException(CopilotException):
```

Comment:
Actually, `RequestException` and `CopilotException` are two completely different types of errors, and there is no inheritance relationship, right?

Reply:
The `CopilotException` is defined as the root exception for all exceptions managed by *Copilot for Azure*. The exceptions describing GPT Error are renamed to GPTException and are children of ServiceException.

Reply:
Yes, I noticed this and I updated the comments before. Could you please take a look again?

Reply:
That makes sense. I will remove the `CopilotException`.